### PR TITLE
CE-4: Hero banner done

### PR DESCRIPTION
**Pull request by jibranmamond**
Hero banner completed.
[CE-4](https://techsang.atlassian.net/browse/CE-4)

### CODE REVIEW

File: assets/css/style.scss [hunk 1]
```diff
@@ -27,23 +36,37 @@ body {
     height: 52px;
 }
 
+.top {
```
Comment (shakilahmad46) [inline assets/css/style.scss:38]: LGTM

File: assets/css/style.css [hunk 2]
```diff
@@ -1,4 +1,10 @@
-body * {
+body {
```
Comment (shakilahmad46) [inline assets/css/style.css:1]: Remove BG Settings for body

**Review by shakilahmad46 — CHANGES_REQUESTED** [review]
Make the changes I requested

File: assets/css/style.css (hunk 2 above)
Comment (jibranmamond) [reply assets/css/style.css:1]: fixed

**Review by shakilahmad46 — APPROVED** [review]
LGTM

**Pull request merged**